frontend/src/pages/Home/components/JoinUsForm/Form.jsx

```diff
@@ -49,16 +174,26 @@ export const JoinUsForm = (props) => {
                   }
                 >
                   <input
-                    placeholder="Name"
-                    id="txt_name"
+                    placeholder="Email ID"
+                    id="txt_email"
                     type="text"
                     required="required"
```

Comment:
as we are doing the validation using joi so I believe we don't need these inbuilt validations

Reply:
actually these were already present in the file before I did anything

Anyways , I will remove them , no issues